src/clamp.js

```diff
@@ -10,14 +10,14 @@ var _curry3 = require('./internal/_curry3');
  * @since v0.20.0
  * @category Relation
  * @sig Ord a => a -> a -> a -> a
- * @param {Number} minimum number
- * @param {Number} maximum number
- * @param {Number} value to be clamped
- * @return {Number} Returns the clamped value
+ * @param {Number} minimum The lower range of the clamp
+ * @param {Number} maximum The upper range of the clamp
```

Comment:
"limit" may be better than "range" here

we should probably indicate if these limits are inclusive or not. `range` for example is inclusive at the bottom, but not at the top.